LICENSE.txt

```diff
@@ -1,22 +1,24 @@
 MIT License
 
-Copyright (c) 2020, Gonzalo Peña-Castellanos
+Copyright (c) 2020- Spyder-Infra Contributors (see AUTHORS.txt)
```

Comment:
```suggestion
Copyright (c) 2020- Spyder-API-Docs Contributors (see AUTHORS.txt)
```

It seems you missed this one when porting the template.

Reply:
Oops; thanks for the catch! The NP++ Find in Files feature refused to find it due to the "ignore case" option not working for some reason, though `git grep` did just fine (I was using a combination of both, but due to some back and forth updating the Infra template to fix issues I found during porting, including one in that very line, it didn't get picked up by that). I doublechecked for other omissions of a similar type, but this was the only one.

I've rebased it in to the appropriate commit (since this, by necessity, is being merged rather than squashed).